tests/src/kakarot/accounts/test_contract_account.py

```diff
@@ -141,6 +141,33 @@ def test_should_read_bytecode(self, cairo_run, bytecode, storage):
             mock_storage.assert_has_calls(calls)
             assert output[:output_len] == list(bytecode)
 
+    class TestNonce:
+        @SyscallHandler.patch("Ownable_owner", 0xDEAD)
+        def test_should_assert_only_owner(self, cairo_run):
+            with cairo_error():
+                cairo_run("test__set_nonce", new_nonce=[])
+
+        @SyscallHandler.patch("Ownable_owner", SyscallHandler.caller_address)
```

Comment:
this is the default right ? so no need to patch

Reply:
@ClementWalter  I just tried deleting @SyscallHandler.patch("Ownable_owner", SyscallHandler.caller_address) but the tests started failing since they need to have an owner. How can I assign an owner without patching?

Reply:
forget my comment sorry, I went to fast, you need to patch